docs/news.rst

```diff
@@ -461,57 +475,59 @@ used to enable it.
    {"_type": "tag", "name": "doIt", "path": "/tmp/foo.py", "pattern": "/^    def doIt():$/", "kind": "member"}
 
 
-See :ref:`Json output <output-json>` for more details.
+See :ref:`JSON output <output-json>` for more details.
 
 
-Changes in tags file format
+Changes to the tags file format
 ---------------------------------------------------------------------
 
 
-Omitting the pattern for too long input line
+Truncating the pattern for long input lines
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
-Not to make too large tags file, a pattern filed of tags file is
-omitted when its size goes beyond 96 bytes. The limit can be
-controlled with ``--pattern-length-limit=N`` option.
+To prevent generating overly large tags files, a pattern field is
+truncated, by default, when its size exceeds 96 bytes. A different
+limit can be specified with ``--pattern-length-limit=N``.
 
-An input source file with single long line causes too large tags file.
-Such input files are popular in javascript: tools for size optimizing
-generate them.
+An input source file with long lines and multiple tag matches per
+line can generate an excessively large tags file with an
+unconstrained pattern length. For example, running ctags on a
+minified JavaScript source file often exhibits this behaviour.
 
 Reference tags
 ---------------------------------------------------------------------
 
-Traditionally ctags collects the information for locating where an
-object having name is DEFINED.
+Traditionally ctags collects the information for locating where a
+language object is DEFINED.
```

Comment:
"language object", this is the term I have been looking for. We should use this term all in the documents. Thank you.